agent/bench-scripts/test-bin/bm

```diff
@@ -8,7 +8,9 @@ echo "$0 $*" >> $_testlog
 if [[ "${_tst_benchmark}" == "pbench-fio" ]]; then
     if [[ "${1%%=*}" == "--client" ]]; then 
         client_file=${1##*=}
-        for client in $(cat ${client_file}); do
+        for line in $(cat ${client_file}); do
+			ip_name="${line%%,*}"
+			client="${ip_name#*:}"
```

Comment:
Indentation?

Reply:
Hmm, isn't `ip_name` used on line 13, and `client` used on line 14?

Reply:
Ah, yes.  (That's what I get for doing reviews too long into the day....)

Reply:
I believe this was addressed in PR #2527.